Player.h

```diff
@@ -1,10 +1,11 @@
-#include "Inventory.h"
+#pragma once
```

Comment:
The `#pragma once` directive usually goes at the very top of the header file, it makes most sense there if you understand how it works (it prevents the compiler from including this header file more than once).